README.md

```diff
@@ -1,11 +1,11 @@
 # Introduction
 
-This repository is the source code for [www.odata.org](http://www.odata.org) built with Jekyll. Site admins regularly updates the contents to the site.
+This repository is the source code for [www.odata.org](https://www.odata.org) built with Jekyll. Site admins regularly updates the contents to the site.
```

Comment:
We cannot support both through re-directing from HTTP to HTTPS?

Reply:
We can. The change for this specific file is optional. The required change is in _config.yml. I figured that I may as well update the popular pages to HTTPS while I am at it.